index.php

```diff
@@ -8,15 +8,19 @@
 Author URI: http://www.jquery.org
 */
 
+define('BASE_URL', 'http://vagrant.learn.jquery.com/jquery-wp-content/plugins/jquery-q-and-a-wp/');
+session_start();
+
 add_action('admin_menu', 'add_option_page');
-add_action( 'admin_enqueue_scripts', 'my_enqueue' );
-add_action( 'wp_ajax_embedQuestion', 'embedQuestion' );
-// add_action( 'wp_ajax_preview_question', 'preview_question' );
-add_action( 'wp_ajax_updatePage', 'updatePage' );
+
+require_once('models/Plugin.php');
+
+//phpinfo();
+register_activation_hook( __FILE__, 'test' );
 
 function add_option_page(){
     enqueue_styles();
-    add_menu_page( 'jQuery Question and Answer', 'jQuery Q & A', 'manage_options', 'jquery-q-and-a', 'init_options_page', 'https://upload.wikimedia.org/wikipedia/commons/f/fa/Question_mark_white_icon.svg');
+    add_menu_page( 'jQuery Question and Answer', 'jQuery Q & A', 'manage_options', 'jquery-q-and-a', 'init_options_page');
```

Comment:
Use `main` instead of `init_options_page`. All the latter is doing is calling `main` anyway.